web/src/components/molecules/MyIntegrations/WebhookForm/index.tsx

```diff
@@ -8,24 +9,78 @@ import Form from "@reearth-cms/components/atoms/Form";
 import Icon from "@reearth-cms/components/atoms/Icon";
 import Input from "@reearth-cms/components/atoms/Input";
 import Row from "@reearth-cms/components/atoms/Row";
+import { WebhookTrigger } from "@reearth-cms/components/molecules/MyIntegrations/types";
 import { useT } from "@reearth-cms/i18n";
-import { validateURL } from "@reearth-cms/utils/regex";
 
-const WebhookForm: React.FC = () => {
+export type Props = {
+  onBack?: () => void;
+  webhookInitialValues?: any;
+  onWebhookCreate: (data: {
+    name: string;
+    url: string;
+    active: boolean;
+    trigger: WebhookTrigger;
+  }) => Promise<void>;
+  onWebhookUpdate: (data: {
+    webhookId: string;
+    name: string;
+    url: string;
+    active: boolean;
+    trigger: WebhookTrigger;
+  }) => Promise<void>;
+};
+
+const WebhookForm: React.FC<Props> = ({
+  onWebhookCreate,
+  onWebhookUpdate,
+  webhookInitialValues,
+  onBack,
+}) => {
   const t = useT();
+
   const [form] = Form.useForm();
 
-  const options: CheckboxOptionType[] = [
-    { label: t("Create"), value: "Create" },
-    { label: t("Update"), value: "Update" },
-    { label: t("Delete"), value: "Delete" },
-    { label: t("Accessed by API"), value: "Accessed by API" },
+  const itemOptions: CheckboxOptionType[] = [
+    { label: t("Create"), value: "onItemCreate" },
+    { label: t("Update"), value: "onItemUpdate" },
+    { label: t("Delete"), value: "onItemDelete" },
+    { label: t("Publish"), value: "onItemPublish" },
+    { label: t("Unpublish"), value: "onItemUnPublish" },
   ];
 
+  const assetOptions: CheckboxOptionType[] = [
+    { label: t("Upload"), value: "onAssetUpload" },
+    { label: t("Delete"), value: "onAssetDeleted" },
+  ];
+
+  const urlRegex =
+    /^(?:(?:https?|ftp):\/\/)(?:\S+(?::\S*)?@)?(?:(?!(?:10|127)(?:\.\d{1,3}){3})(?!(?:169\.254|192\.168)(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,}))\.?)(?::\d{2,5})?(?:[/?#]\S*)?$/i;
```

Comment:
Should be using the utility you created a couple PRs ago

Reply:
I guess this happened because of a conflict, I'm not sure.
Fixed, Now using utils function.